doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc

```diff
@@ -54,12 +53,14 @@ complete the following additional steps:
 * *Enable SSL integration*: Select this setting if you want to use an SSL connection to the AMQ broker. In this case you must also provide the name of the secret that you created in <<secrets-amq-create-proc>> and the names and passwords of the key store and trust store that you used for the secret. 
 . If you want to customize the configuration of the Java virtual machine on the {KIE_SERVER} pods, select the *Enable JVM configuration* box and then enter information in any of the fields under *Enable JVM configuration*. All fields are optional. For the JVM parameters that you can configure, see <<jvm-settings-ref>>.
 ifdef::PAM[]
-. Select the database that the {KIE_SERVER} must use. The following values are available:
+. In the *Database type* field, select the database that the {KIE_SERVER} must use. The following values are available:
 ** `mysql`: A MySQL server, created in a separate pod.
 ** `postgresql`: A PostgreSQL server, created in a separate pod. Use this setting unless you have a specific reason to use any other setting.
 ** `h2`: A built-in `h2` database engine that does not require a separate pod. Do not scale the {KIE_SERVER} pod if you use this setting.
 ** `external`: An external database server. 
-. If you selected any database except `external`, a Persistent Volume Claim will be created to store the database. Optionally, in the *Size* field, enter the size of the persistence volume to create.
+. If you selected any database except `external`, a Persistent Volume Claim will be created to store the database. Optionally, set configuration parameters of the persistent volume to create:
```

Comment:
Suggestion - move "optionally" and resolve sentence fragment "to create":

Optionally, if you selected any database except `external`, a Persistent Volume Claim is created to store the database and you must set the configuration parameters for the persistent volume: